Dockerfile

```diff
@@ -1,12 +1,8 @@
 FROM jujusolutions/jujubox:latest
+MAINTAINER Charles Butler <charles.butler@canonical.com>
 
-VOLUME "/home/ubuntu/.juju"
-VOLUME "/home/ubuntu/trusty"
-VOLUME "/home/ubuntu/builds"
-VOLUME "/home/ubuntu/layers"
-VOLUME "/home/ubuntu/interfaces"
-ADD install-review-tools.sh /install-review-tools.sh
-RUN /install-review-tools.sh
-# Override jujubox run.sh
-ADD run.sh /run.sh
-CMD /run.sh
+ENV LAYER_PATH=/home/ubuntu/charms/layers
```

Comment:
The layer and interface path may be different based on the build argument $JUJU_USER in jujubox. I do not suspect we can "read" that argument from a different build, but I think we should allow this one to be created with another user as well.

Oh wait! That user stuff has not landed! Nevermind.